src/languageSupport/languages/flux/lsp/monaco.flux.lsp.ts

```diff
@@ -71,8 +72,10 @@ export function initLspWorker() {
     return
   }
   if (window.Worker) {
+    event('Used LSP worker.')
     worker = new Worker(fluxWorkerUrl)
   } else {
+    event('Used LSP fallback worker.')
```

Comment:
Can we send any other additional information? I'd like to know what browser it is that they are using. Maybe also the org id or something? I think there's value in understanding that user's use case, so maybe Product can learn more about the user.

Reply:
Metadata is collected with the event, including platform (e.g. web) and browser.